boards/shields/xenvm_dom0/Kconfig.defconfig

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_XENVM_DOM0
+
+config NUM_IRQS
+	int
+	default 500
+
+config MAX_XLAT_TABLES
+	default 24
+
+config HEAP_MEM_POOL_SIZE
+	default 2097152
+
+# Enable PV hypervisor console
+config UART_XEN_HVC_CONSOLEIO
+	default y
+
+config TEST
```

Comment:
Do you really need to enable this for all builds?

Reply:
sorry, I didn't investigate this and below questions, I have just copy-pasted a part of XENVM board config